net.wz2100.wz2100.yaml

```diff
@@ -35,25 +35,25 @@ modules:
     - -DPHYSFS_BUILD_TEST=OFF
     sources:
       - type: archive
-        url: 'http://icculus.org/physfs/downloads/physfs-3.0.1.tar.bz2'
-        sha1: 9959a7acad0aa30d1a86d3a418e61155b0ac533c
+        url: 'http://icculus.org/physfs/downloads/physfs-3.0.2.tar.bz2'
+        sha1: 6a15c458898d0570101d8f173201cde9ced78df7
       
   - name: fribidi
     config-opts:
     - --disable-docs
     sources:
       - type: git
         url: https://github.com/fribidi/fribidi
-        tag: v1.0.5
-        commit: 5b6a16e8da12ae7ff482fbfa5a17b72bd518418f
+        tag: v1.0.7
+        commit: aac3845708969b9c4bc2874edc4950123595b1b4
```

Comment:
Fribidi has updated to version v1.0.8 while waiting for someone to merge this pull request, unfortunately.

Reply:
Will merge this. Mind sending a merge request to update Fribidi? Thank you